.github/workflows/gems_ci.yml

```diff
@@ -14,21 +14,21 @@ env:
   ruby_version: 3.3
 
 jobs:
-  rspec:
+  test:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
         ruby: [3.0] #, 3.1, 3.2, 3.3]
```

Comment:
we should probably be consistent with which ruby version?  doesn't really matter for now, but maybe just stick with 3.3?